narwhals/functions.py

```diff
@@ -52,7 +52,7 @@ def from_dict(
     data: dict[str, Any],
     schema: dict[str, DType] | Schema | None = None,
     *,
-    native_namespace: ModuleType,
+    native_namespace: ModuleType | None = None,
```

Comment:
I am somewhat scared to provide a default, but also it should break if incompatible, so maybe no harm in that

Reply:
why scared? would you prefer if people pass `nw.from_dict(data, native_namespace=None)` explicitly?

Reply:
Fair point 👌